src/analysis/clustering.jl

```diff
@@ -22,7 +24,7 @@ Vector{Float64}:
  5
 """
 function _complexity(x::AbstractMatrix{T})::Vector{Float64} where {T<:Real}
-    return vec(sqrt.(sum(diff(Matrix(x), dims=1) .^ 2, dims=1)) .+ 1)
+    return vec(sqrt.(sum(diff(Matrix(x); dims=1) .^ 2; dims=1)) .+ 1)

```

Comment:
`T` could be removed for conciseness as it is only used once 

(can't suggest code change as the line wasn't modified in this PR)